scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja

```diff
@@ -60,6 +62,16 @@
 {% endfor %}
 /* END-driver id */
 
+/* BEGIN-Common Macro definitions */
+{% macro entry_point_name(capability, entry_point, driver) -%}
```

Comment:
Looking at this macro definition: why isn't it a Python function? It doesn't seem to be making use of any actual templating capabilities beyond one string concatenation.

Generally, as much intelligence as possible should be in Python rather than inside the templates, since Python is easier to read, easier to debug and easier to document.

Reply:
It's called as a template from both the One Shot templates. Since it is called from both places we made it a macro, If I recollect this right, it was made into a macro based on the review comments.